plugins/module_utils/gitlab.py

```diff
@@ -7,11 +7,11 @@
 from __future__ import (absolute_import, division, print_function)
 __metaclass__ = type
 
-from distutils.version import StrictVersion
-
 from ansible.module_utils.basic import missing_required_lib
 from ansible.module_utils.common.text.converters import to_native
 
+from ansible_collections.community.general.plugins.module_utils.version import LooseVersion
```

Comment:
```suggestion
from ansible_collections.community.general.plugins.module_utils.version import LooseVersion
```
`StrictVersion` is also present in `ansible.module_utils.compat.version`. Is this replacement intended?

Reply:
Yes. There is no reason to use `StrictVersion` when `LooseVersion` also suffices.

Reply:
OK, thanks for clarifying:)